Dockerfile

```diff
@@ -25,18 +27,15 @@ RUN ln -s /etc/apache2/sites-available/rc-index.conf /etc/apache2/sites-enabled/
  && rm -f /etc/apache2/sites-enabled/000-default.conf
 
 RUN a2enmod rewrite \
- && a2enmod php5 \
- && ln -s /etc/php5/mods-available/mcrypt.ini /etc/php5/apache2/conf.d/20-mcrypt.ini \
- && ln -s /etc/php5/mods-available/mcrypt.ini /etc/php5/cli/conf.d/20-mcrypt.ini \
  && mkdir -p /var/container_init \
  && mkdir -p /var/www/rc-index \
  && mkdir -p /var/log/supervisor \
  && mkdir -p /var/www/rc-index/logs
 
 # Configure PHP
-RUN sed -i "s/^post_max_size =.*/post_max_size = 5120M/" /etc/php5/apache2/php.ini \
- && sed -i "s/^upload_max_filesize =.*/upload_max_filesize = 5120M/" /etc/php5/apache2/php.ini \
- && sed -i "s/^memory_limit =.*/memory_limit = 256M/" /etc/php5/apache2/php.ini
+#RUN sed -i "s/^post_max_size =.*/post_max_size = 5120M/" /etc/php5/apache2/php.ini \
```

Comment:
We should remove this commented code to improve the readability of the file. We can always look back in source control if we want to recover it.